src/algebra/order/to_interval_mod.lean

```diff
@@ -732,4 +732,26 @@ end
 lemma to_Ico_mod_zero_one (x : α) : to_Ico_mod (0 : α) zero_lt_one x = int.fract x :=
 by simp [to_Ico_mod_eq_add_fract_mul]
 
+lemma Union_Ioc_int_cast : (⋃ (n : ℤ), set.Ioc (n:α) (n + 1)) = set.univ :=
```

Comment:
I think it would be worth proving the more general:
```suggestion
lemma Union_Ioc_zsmul (a : α) (ha : a \ne 0) : (⋃ (n : ℤ), set.Ioc (n • a) ((n + 1) • a) = set.univ :=
```
and then specializing to the version you currently have with `simpa using Union_Ioc_zsmul 1`